src/electron/platform/android/setup/android-setup-start-listener.ts

```diff
@@ -11,14 +11,14 @@ export class AndroidSetupStartListener {
         private readonly androidSetupActionCreator: AndroidSetupActionCreator,
     ) {}
 
-    public initialize(): void {
+    public async initialize(): Promise<void> {
         this.userConfigStore.addChangedListener(this.storeChanged);
         this.androidSetupStore.addChangedListener(this.storeChanged);
 
-        this.storeChanged();
+        await this.storeChanged();
     }
 
-    private storeChanged = () => {
+    private storeChanged = async () => {
```

Comment:
Instead of making this async, could it just be wrapped in an async arrow function where it needs to be passed as an async function? It looks like it's registered as a sync callback for AndroidSetupStore, so the promise is probably being floated there

Reply:
That change would also let initialize() stay as sync, so this PR wouldn't touch as many electron files